src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,24 @@
+package main.java.commands;
+
+import main.java.commands.Command;
+import main.java.tasks.*;
+import main.java.tasklist.*;
```

Comment:
Perhaps you could import each class that you would require explicitly instead of using wildcard imports?